lib/improver/nbhood.py

```diff
@@ -428,15 +462,50 @@ def __init__(self, neighbourhood_method, radii_in_km, lead_times=None,
                        "Unable to continue due to mismatch.")
                 raise ValueError(msg)
         self.unweighted_mode = bool(unweighted_mode)
+        self.ens_factor = float(ens_factor)
+
+    def _find_radii(self, num_ens, cube_lead_times=None):
+        """Revise radius/radii for found lead times and ensemble members
+
+        If cube_lead_times is None just adjust for ensemble
+        members if necessary.
+        Otherwise interpolate to find radius at each cube
+        lead time and adjust for ensemble members if necessary.
+
+        Parameters
+        ----------
+        num_ens : float
+            Number of ensemble members / realizations.
```

Comment:
There are lots of places in this code where / has been used instead of or e.g. x/y and time/forecast_reference_time. I've replaced them where I can but some would require modifications to the unit tests in code I haven't written so I've left those in. If you feel strongly about these please can you raise technical debt ticket to have these corrected.